app/views/username/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: dashboard_path)
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Edit your username</h1>
+
+    <%= form_with model: current_user, url: username_path do |f| %>
+      <%= f.govuk_text_field :username, label: { text: "Username" }, width: "two-thirds" %>
```

Comment:
Do we need label text here? Seeing how it's similar to the name, maybe the gem does it for us.

Reply:
We could make the title the label - I was just following the designs as I don't really have a strong opinion about this

Reply:
I mean, does 
```
      <%= f.govuk_text_field :username, width: "two-thirds" %>
```
produce the same result as current code?

Reply:
Ah sorry I misunderstood your original comment - I didn't realise the gem fills it in for us.

I think it's better to be verbose, especially if we end up changing what we call it internally (or externally) to preferred name or something like that